plugins/org.eclipse.elk.alg.common/src/org/eclipse/elk/alg/common/nodespacing/internal/algorithm/NodeLabelAndSizeUtilities.java

```diff
@@ -229,4 +373,31 @@ public static boolean isFirstOutsidePortLabelPlacedDifferently(final NodeContext
         }
     }
     
+    /**
+     * Compute the bounds of the labels of the port corresponding to the {@code portContext}. This bounds is relative to
```

Comment:
Typo: _These bounds_

Reply:
"bounds" will be replaced by "the bounding box"